source/blueprints/banking.rst

```diff
@@ -0,0 +1,546 @@
+    Selecting hyperparameters using k-fold cross-validation with 5 splits
+    Best accuracy: 95.92%, params: {'C': 10, 'class_weight': {0: 0.8992592592592593, 1: 1.2360330578512397, 2: 0.8992592592592593, 3: 0.8033777777777779, 4: 1.6324705882352941}, 'fit_intercept': False}
+
+Change the classification model to random forest instead of the default logistic regression:
+
+.. code:: python
+
+   ic.fit(model_settings={'classifier_type': 'rforest'}, param_selection={'type': 'k-fold', 'k': 10, 'grid': {'class_bias': [0.7, 0.3, 0]}})
+
+.. code-block:: console
+
+    Fitting intent classifier: domain='accounts_creditcards'
+    Selecting hyperparameters using k-fold cross-validation with 10 splits
+    Best accuracy: 92.46%, params: {'class_weight': {0: 1.0, 1: 1.0, 2: 1.0, 3: 1.0, 4: 1.0}}
+
+You can use similar options to inspect and experiment with the Entity Recognizer and the other NLP classifiers. Finding the optimal machine learning settings is a highly iterative process involving several rounds of model training (with varying configurations), testing, and error analysis. See the User Guide for more about training, tuning, and evaluating the various MindMeld classifiers.
+
+.. admonition:: Exercise
+
+   Experiment with different models, features, and hyperparameter selection settings to see how they affect classifier performance. Maintain a held-out validation set to evaluate your trained NLP models and analyze misclassified test instances. Then, use observations from the error analysis to inform your machine learning experimentation. See the :doc:`User Guide <../userguide/nlp>` for examples and discussion.
+
+
+9. Parser Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The relationships between entities in the banking assistant queries are simple ones. For example, in the annotated query ``can you pay my {minimum|credit_amount} bill``, the ``credit amount`` entity is self-sufficient, in that it is not described by any other entity.
+
+If you extended the app to support queries with more complex entity relationships, it would be necessary to specify *entity groups* and configure the parser accordingly. For example, in the query ``can you pay my {minimum|credit_amount} bill for my {amex|credit_card}``, we would need to relate the ``credit_amount`` entity to the ``credit_card`` entity, because one entity describes the other. The related entities would form an entity group. For more about entity groups and parser configurations, see the :doc:`Language Parser <../userguide/parser>` chapter of the User Guide.
+
+Since we do not have entity groups in the banking assistant, we do not need a parser configuration.
+
+
+10. Using the Question Answerer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :doc:`Question Answerer <../userguide/kb>` component in MindMeld is mainly used within dialogue state handlers for retrieving information from the knowledge base. Since the banking assistant has no knowledge base, no question answerer is not needed.
```

Comment:
```suggestion
The :doc:`Question Answerer <../userguide/kb>` component in MindMeld is mainly used within dialogue state handlers for retrieving information from the knowledge base. Since the banking assistant has no knowledge base, question answerer is not needed.
```